README.md

```diff
@@ -25,26 +36,40 @@ $ npm install -g @filecoin-station/core
 $ FIL_WALLET_ADDRESS=f1... station
 ```
 
-## Configuration
+## Common Configuration
 
-- `$XDG_STATE_HOME` Station stores logs and module state in
-`$XDG_STATE_HOME/filecoin-station`. Defaults to `~/.local/state`.
+Station Core is configured using environment variables (see
+[The Twelve-Factor App](https://12factor.net/config)).
+
+The following configuration options are shared by all Station commands:
+
+- `$XDG_STATE_HOME` _(string; optional)_: Station stores logs and module state
+  in the directory `$XDG_STATE_HOME/filecoin-station`. State home defaults to
+  `~/.local/state`.
 
 ## Commands
 
 ### `$ station`
 
-Start Station.
+Start a new Station process. The Station will run in foreground and can be
+terminated by pressing Ctrl+C.
+
+This command has the following additional configuration in addition to common
+the configuration options described in
+[Common Configuration](#common-configuration):
 
-Configuration:
+- `FIL_WALLET_ADDRESS` _(string; required)_: Address of the Filecoin wallet that
+  will receive rewards. The value must be a mainnet address starting with `f1`.
 
-- `$FIL_WALLET_ADDRESS` Filecoin wallet address (required). If you just want
-to give `core` a quick spin, you can use the address
-`f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za`. Please note that any earnings
-sent there will be lost.
+  If you just want to give `core` a quick spin, you can use the address
+  `f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za`. Please note that any earnings
+  sent there will be lost.
 
 ### `$ station metrics`
 
+_This command reads information from the state files created by the main Station
```

Comment:
"main Station process" is a term not yet established. Also I don't think we need to talk about files here, does the user care where that data is coming from? To me this is too much an implementation detail. Therefore I wonder what you're trying to achieve by adding this text 🤔

Reply:
I am so glad you started this discussion, @juliangruber 🙏🏻

Reply:
Hmm, if you compare this to `docker`, the `logs` command doesn't care if the container is running or not. I agree though that we should prevent concurrent `$ station` calls!

To me, annotating that logs are outdated doesn't make sense. The logs won't be outdated, they will be as uptodate as they can be. Only nothing is running that can produce new logs.

Can you maybe think of another CLI that works the way you propose?

Reply:
> Maybe we can also improve all commands to print information about the location of state files at startup to make troubleshooting easier.

Are you sure this is a problem? Have you run into this?

Reply:
> If you don't have the main Station Core process running, there will be either no metrics and no logs, or the data will be outdated.

To me it's obvious that there will be no new metrics or logs if the thing isn't running. But maybe I have a blind spot? It's the same with systemd: You can always get the logs. If you don't get new logs, you check if the service is running, and otherwise start it.